Sources/SpeziDevices/SpeziDevices.docc/SpeziDevices.md

```diff
@@ -0,0 +1,41 @@
+# ``SpeziDevices``
+
+<!--@START_MENU_TOKEN@-->Summary<!--@END_MENU_TOKEN@-->
+
+<!--
+
+This source file is part of the Stanford Spezi open-source project
+
+SPDX-FileCopyrightText: 2024 Stanford University and the project authors (see CONTRIBUTORS.md)
+
+SPDX-License-Identifier: MIT
+
+-->
+
+## Overview
+
+Below is a list of important symbols of SpeziDevices.
```

Comment:
A visual code example like the one in the inline comments would be great here. Some hints pointing to the UI might be very helpful here.